tests/unit/amo/components/TestAddonReviewCard.js

```diff
@@ -330,7 +496,7 @@ describe(__filename, () => {
       }),
     );
 
-    const root = render({ review });
+    const root = render({ review, smallCard: true });
```

Comment:
Why `smallCard: true`? Doesn't the message also appear when `smallCard === false`?